packages/hash/api/src/graphql/typeDefs/knowledge/comment.typedef.ts

```diff
@@ -84,6 +93,20 @@ export const persistedCommentTypedef = gql`
       tokens: [TextToken!]!
     ): PersistedComment!
 
+    """
+    Update an existing comment
+    """
+    updatePersistedComment(
+      """
+      Id of the comment being deleted
+      """
+      entityId: ID!
+      """
+      Id of the comment being deleted
```

Comment:
```suggestion
      The properties to update
```

Reply:
Yeah my bad I first decided on doing a deletion-specific mutation and when I changed with a general update properties mutation I forgot to update the annotations. Updated the annotations in e54545814ea2ab44fa446fadc368585a1f40cd9f.